benchexec/tools/divine4.py

```diff
@@ -86,16 +94,17 @@ def determine_result(self, returncode, returnsignal, output, isTimeout):
         and should give some indication of the failure reason
         (e.g., "CRASH", "OUT_OF_MEMORY", etc.).
         """
+        run.exit_code.value = run.exit_code.value or 0
```

Comment:
This will not work, the field should be immutable.

Reply:
fixed in 53d68f15f868f1dca4a307f032076a0ddc5a43c3